client/src/components/_common/TextCopyField/TextCopyField.jsx

```diff
@@ -34,22 +34,16 @@ const TextCopyField = ({ value, placeholder }) => {
       <div className="input-group-prepend">
         <CopyToClipboard text={value}>
           <Button
-            className={`${styles['copy-button']} ${
-              isCopied ? styles['is-copied'] : ''
-            }`}
             // RFE: Avoid manual JS ↔ CSS sync of transition duration by using:
             //      - `data-attribute` and `attr()` (pending browser support)
             //      - PostCSS and JSON variables (pending greater need for this)
-            style={{ '--transition-duration': `${transitionDuration}s` }}
+            //style={{ '--transition-duration': `${transitionDuration}s` }}
```

Comment:
Is this feature changed because <Button> does not support `className`? (a bug—easy to fix)

[I ask Design privately whether [users need] the previous UX.](https://tacc-team.slack.com/archives/G01K9ET7JM7/p1657837094117899)

<details>

https://user-images.githubusercontent.com/62723358/179099012-abbf2ac7-28dd-485e-9192-d57cb790f6d5.mov

https://user-images.githubusercontent.com/62723358/179099016-488d3344-ee1d-4a90-89e7-3ce983c63924.mov

</details>

Reply:
**Resolution**:
The new UI is acceptable. Please remove artifacts of the old UI (e.g. `transitionDuration`, `--transition-duration`).